e2e_test/source/cdc/cdc.load.slt

```diff
@@ -242,3 +242,20 @@ create table enum_to_varchar (
  table.name = 'enum_table',
  slot.name = 'enum_to_varchar'
 );
+
+
+statement ok
+create table orders (
+  id int,
+  name varchar,
+  PRIMARY KEY (id)
+) with (
+ connector = 'postgres-cdc',
+ hostname = '${PGHOST:localhost}',
+ port = '${PGPORT:5432}',
+ username = '${PGUSER:$USER}',
+ password = '${PGPASSWORD:}',
+ database.name = '${PGDATABASE:postgres}',
```

Comment:
BTW, we better to test whether uppercase in `database.name` can work.

Reply:
Tested. It works. But I also found a weird bug https://github.com/risingwavelabs/risingwave/pull/17754#issuecomment-2240873767. After some attempts, it's sometimes not reproduciable: only several special tables can trigger this bug. I'm still not aware of the exact condition. To merge this PR, I temporarily delete the tests about `database.name`.

Reply:
ok, we didn't get bug report from our users yet. Agree to create a issue to track it.